assets/js/base/components/quantity-selector/index.js

```diff
@@ -1,8 +1,14 @@
 /**
  * External dependencies
  */
+import { __, sprintf } from '@wordpress/i18n';
+import { speak } from '@wordpress/a11y';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import { useRef, useState, useEffect, useCallback } from '@wordpress/element';
+
+const KEYCODE_UP = 38;
+const KEYCODE_DOWN = 40;
```

Comment:
`keyCode` is deprecated, I'm not sure if we should be using it.
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode
We should use `code` instead.
```js
const KEYCODE_UP = 'ArrowUp';
const KEYCODE_DOWN = 'ArrowDown';
```

Reply:
Unfortunately, `code` isn't supported on [Internet Explorer or Edge](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code#Browser_compatibility). Since we still support those browsers (gut check, do we?), we might have to use progressive enhancement here. If `code` is supported in the current browser, use it, otherwise use `keyCode`.

Reply:
https://caniuse.com/#feat=keyboardevent-code Latest Edge does. We can detect it, otherwise fallback to keyCode.